docs/modules/ROOT/pages/create-github-action.adoc

```diff
@@ -338,18 +338,20 @@ The token is required when consuming the GitHub REST or GraphQL APIs so, in most
 [[private_repo]]
 [WARNING]
 ====
-The workflow above will malfunction when used in a private repository. The automatically generated `GITHUB_TOKEN` only grants access to packages, that are part of the https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages#access-tokens[same repository] the workflow is defined in.
+You may call a Quarkus GitHub Action defined in a public repository from a public or private GitHub repository.
 
-To use a _GitHub Action_ defined in a separate private repository, belonging to the same user or organization, please follow these steps:
```

Comment:
> belonging to the same user or organization

I think once you have a PAT set up, it will work for any repo for which you have the permission even if cross orgs?

Reply:
I'm not sure, but I can drop the subclause.

Reply:
On second thought, it may be possible to access packages from other users, but actions in private repositories can only be accessed from within the same organisation, provided they are set accessible.